data/locales/ba.lua

```diff
@@ -0,0 +1,86 @@
+return {
+	-- Buttons, context menu
+	['ui_use'] = "Iskoristi",
+	['ui_give'] = "Daj",
+	['ui_close'] = "Zatvori",
+	['ui_drop'] = "Baci",
+	['ui_removeattachments'] = "Skini Dodatke",
+	-- Tooltip
+	['ui_durability'] = "Izdržljivost",
+	['ui_ammo'] = "Metci",
+	['ui_serial'] = "Serijski Broj",
+	['ui_components'] = "Komponente",
+	['ui_tint'] = "Tinta",
+	-- Useful controls
+	['ui_usefulcontrols'] = "Korisne kontrole",
+	['ui_rmb'] = "Item Contex Menu",
+	['ui_ctrl_lmb'] = "Brzo premjestite hrpu predmeta u drugi inventar",
```

Comment:
Here you use "predmeta" but in most cases you use "item" so should really decide between the two

I think "predmet" is probably a better translation all around